src/cms/enhanced-media-gallery.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Enhanced Media Gallery
+---
+
+Merchants can use the Enhanced Media Gallery to organize and manage their media files on the server. The Enhanced Media Gallery contains the same functionalities as the Media Storage with an improved user interface and a closer integration with [Adobe Stock][adobe-stock].
+
+![Enhanced Media Gallery Grid]({% link images/images/enhanced-media-gallery/gallery-grid.png %}){: .zoom}
+
+## Prerequisites
+
+The Enhanced Media Gallery requires Magento 2.4 or newer. 
+
+## Enable the Enhanced Media Gallery
```

Comment:
Please add the information covering the:
 - Assets and asset-content relations synchronization and corresponding CLI commands
 - View details slide panel functionality (covering the information and actions, ensure to feature the "Used In" information that is new to the Magento admin user)

Reply:
As discussed the CLI commands will be addressed on the DevDocs and we will link this later.